ios/Sources/Component/SearchController.swift

```diff
@@ -5,29 +5,32 @@ public class SearchController: NSObject {
 
     public let wrappedValue = UISearchController()
     private let searchTextDidChangeTo: (String) -> Void
-    private let willDismissSearchController: () -> Void
+    private let isSearchTextEditing: (Bool) -> Void
 
     public init(
         searchBarPlaceHolder: String? = nil,
         searchTextDidChangeTo: @escaping (String) -> Void,
-        willDismissSearchController: @escaping () -> Void
+        isSearchTextEditing: @escaping (Bool) -> Void
     ) {
         self.searchTextDidChangeTo = searchTextDidChangeTo
-        self.willDismissSearchController = willDismissSearchController
+        self.isSearchTextEditing = isSearchTextEditing
         super.init()
         wrappedValue.searchBar.delegate = self
-        wrappedValue.delegate = self
+        // W/A first time showing search result, black blur view does not work correctly
+        wrappedValue.obscuresBackgroundDuringPresentation = false
     }
 }
 
 extension SearchController: UISearchBarDelegate {
     public func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {
         searchTextDidChangeTo(searchText)
     }
-}
 
-extension SearchController: UISearchControllerDelegate {
-    public func willDismissSearchController(_ searchController: UISearchController) {
```

Comment:
I deleted this because `searchBarTextDidBeginEditing` called later than `willDismissSearchController` and better name with using `searchBarTextDidBeginEditing`